.github/workflows/benchmarks.yml

```diff
@@ -20,45 +20,21 @@ jobs:
     name: Benchmarks
     runs-on: benchmark
     timeout-minutes: 30
-    strategy:
-      fail-fast: false
-      max-parallel: 4
-      matrix:
-        python-version: [ 3.9 ]
+    container: "docker://ubuntu:impish"
 
     steps:
+    - name: Prepare Ubuntu
+      env:
+        DEBIAN_FRONTEND: noninteractive
+      run: |
+        apt-get --yes update
+        apt-get install --yes git lsb-release sudo
+
     - name: Checkout Code
       uses: actions/checkout@v3
       with:
         fetch-depth: 0
 
```

Comment:
- name: Checkout test blocks and plots
      uses: actions/checkout@v3
      with:
        repository: 'Chia-Network/test-cache'
        path: '.chia'
        ref: '0.28.0'
        fetch-depth: 1